### ISSUE TITLE
onLeft and leftTitle do not work using Example on tab2_2

### ISSUE THREAD

**Issue by esutton** [aksonov/react-native-router-flux#789]
### Version
0.26 is required for v3): 

- react-native-router-flux v3.26.16
- react-native v0.26

### Expected behaviour
- Expected left button to display "Left"
- Expected left button press to display alert "Left button!"

File: Example.js
```
<Scene key="tab2" initial={true} title="Tab #2" icon={TabIcon}>
  <Scene key="tab2_1" component={TabView} title="Tab #2_1" renderRightButton={()=><Right/>} />
  <Scene key="tab2_2" component={TabView} title="Tab #2_2" onLeft={()=>alert("Left button!")} leftTitle="Left" duration={1} panHandlers={null}/>
</Scene>
```

### Actual behaviour
- The **Tab #2_2** left button displayed no text, only the left arrow
- **Tab #2_2**  scene transitioned to tab2_1 without displaying "Left button!" alert

### Steps to reproduce

0. cd Examples, run "npm install"
1. Run react-native-router-flux/Example in iOS simulator, react-native run-ios
2. Select "Go to TabBar page"
3. Select "next screen for tab2_1"
4. Title displays "Tab #2_2" with no left button text displayed

![screen shot 2016-06-06 at 2 39 44 pm](https://cloud.githubusercontent.com/assets/6115047/15835215/87b42776-2bf4-11e6-8f07-4c72f860b09a.png)

**Comment by aksonov** [comment]
Should be fixed with 3.26.20

**Comment by aksonov** [comment]
3.26.21 reverts Back button over Left button priority, because it is more correct (don't show drawer but back button). In rare case you could use onBack, backTitle and new hideBackImage properties to show something custom instead of own back button
